build.sbt

```diff
@@ -408,8 +412,15 @@ lazy val noPublishSettings = Seq(
 )
 
 lazy val mimaSettings = mimaDefaultSettings ++ Seq(
-  failOnProblem := compatibleVersion(version.value).isDefined,
-  previousArtifact := compatibleVersion(version.value) map {
-    organization.value % s"${moduleName.value}_${scalaBinaryVersion.value}" % _
-  }
+  failOnProblem <<= version(compatibleVersion(_).isDefined),
+  previousArtifact <<= (version, organization, scalaBinaryVersion, moduleName)((ver, org, binVer, mod) => compatibleVersion(ver) map {
+    org % s"${mod}_${binVer}" % _
+  })
 )
+
+// Check whether to enable java 8 type lambdas
+// https://github.com/scala/make-release-notes/blob/2.11.x/experimental-backend.md
+def delambdafyOpts(v: String): Boolean = VersionNumber(v).numbers match {
+  case Seq(2, 11, x, _*) if x > 6 => true
```

Comment:
This will fail someday on Scala 3.x and Scala 2.12.x.  Which isn't important today, but makes me wonder if VersionNumber has an ordering.